website/scripts/index_search_content.js

```diff
@@ -15,30 +17,36 @@ async function main() {
   const pagesFolder = path.join(__dirname, '../pages')
 
   // Grab all search-indexable content and format for Algolia
-  const searchObjects = glob
-    .sync(path.join(pagesFolder, '**/*.mdx'))
-    .map((fullPath) => {
-      const { content, data } = matter.read(fullPath)
-
-      // Get path relative to `pages`
-      const __resourcePath = fullPath.replace(`${pagesFolder}/`, '')
-
-      // Use clean URL for Algolia id
-      const objectID = __resourcePath.replace('.mdx', '')
-
-      const searchableDimensions = Object.keys(data)
-        .filter((key) => SEARCH_DIMENSIONS.includes(key))
-        .map((dimension) => ({
-          [dimension]: data[dimension],
-        }))
-
-      return {
-        ...searchableDimensions,
-        content,
-        __resourcePath,
-        objectID,
-      }
-    })
+  const searchObjects = await Promise.all(
+    glob
+      .sync(path.join(pagesFolder, '**/*.mdx'), {
+        ignore: path.join(pagesFolder, 'partials/**/*'),
```

Comment:
Needed to avoid indexing `.mdx` "partial" files

Reply:
QQ: Will content of included partial files be indexed for the pages that include them ?

Reply:
I think they won't. I tried searching for something that I know that it's in a partial and the search couldn't find it.

Reply:
We should definitely look into this further - we could potentially resolve partials before sending data for indexing if this is the case

Reply:
@azr We currently only send `page_title`, `description` (from the frontmatter) and headlines through to Algolia. In other words, the actual content of the `.mdx` files aren't indexed/searchable. This means we'd only need to introspect the partial files if they contain headings. As @jescalan mentioned, we'd likely want to resolve the partials and perform the index as to not miss indexable headlines.

Reply:
Nice, thanks for the explanation !